site/tests/app/controllers/AuthenticationControllerTester.php

```diff
@@ -13,6 +13,7 @@
 
 /**
  * @runTestsInSeparateProcesses
+ * @preserveGlobalState disabled
```

Comment:
What about these changes is causing something within global state to not be serializable? Is there anyway to avoid that?

Reply:
William and I both spent a couple hours trying to figure out what was going on here. Bumping DBAL is what is causing this issue. They added a closure as a member variable of their [Configuration object](https://github.com/doctrine/dbal/blob/cbc2e57cf9806072c022f4e537b0d9c26dc0a254/src/Configuration.php#L39-L44). Somehow this gets saved into some global state we store and when moving to a separate process, the serialization fails since PHP can't serialize closures. The only 2 feasible options I have found is removing the testing in separate processes (wasn't exactly sure why we are doing this besides memory/isolation reasons?) or by adding this preserve global state option. If you have any other ideas here, please let me know and I'll try them.